ZeroWasteFriendly/Scenes/Library/LibraryWorker.swift

```diff
@@ -26,6 +26,10 @@ final class LibraryWorkerImpl {
 extension LibraryWorkerImpl: LibraryWorker {
 
     func fetchPosts(completion: FetchPostsCompletion?) {
-        repository.fetchLibrary(completion: completion)
+        if R.string.localizable.language() == "Polish" {
```

Comment:
that looks like a code that you'd reuse in many places

1). Do not use names like Polish or English, use codes like: pl-PL or en_UK etc, identifiers we're using for Locale()
2). Make a helper class that does not return String Types, it's a great place for a an enum